designs/UI/css/main.css

```diff
@@ -0,0 +1,77 @@
+@import url(https://fonts.googleapis.com/css?family=Oxygen);
+body {
+    font-size: 12;
+    font-family: 'Oxygen', sans-serif !important;
+}
+.small-text {font-size: 10px;}
+.medium-text {font-size: 12px;}
+.button {
+    background-color: #4CAF50;
+    border: none;
+    color: white;
+    padding: 5px 15px 5px 15px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 14px;
+    font-weight: bold;
+    margin: 4px 2px;
+}
+.border {
+	padding: 20px;
```

Comment:
It's advised to ident your work, you can try linters to auto-indent your work

Reply:
ok